metadata-load/src/test/resources/AnalysisDocumentApi.xml

```diff
@@ -1,49 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why have all the analysis test files been deleted? They will be useful for future tests.